e2e-tests/utils.sh

```diff
@@ -131,19 +131,24 @@ function retry_times() {
 }
 
 function check_operator_log_for_errors {
+  local ignore=$1
   echo "Checking for operator log errors..."
   operator_pod_namespace=$(get_operator_pod_namespace)
   operator_pod_name=$(get_operator_pod_name)
   echo "Operator namespace: ${operator_pod_namespace} pod: ${operator_pod_name}"
-  errors=$(kubectl logs -n "${operator_pod_namespace}" "${operator_pod_name}" \
-      | grep -v "Exception while listing jobs" `#https://issues.apache.org/jira/browse/FLINK-30146` \
-      | grep -v "Failed to submit a listener notification task" `#https://issues.apache.org/jira/browse/FLINK-30147` \
-      | grep -v "Failed to submit job to session cluster" `#https://issues.apache.org/jira/browse/FLINK-30148` \
-      | grep -v "Error during event processing" `#https://issues.apache.org/jira/browse/FLINK-30149` \
-      | grep -v "REST service in session cluster is bad now" `#https://issues.apache.org/jira/browse/FLINK-30150` \
-      | grep -v "AuditUtils" `#https://issues.apache.org/jira/browse/FLINK-30151` \
-      | grep -v "Error while patching status" `#https://issues.apache.org/jira/browse/FLINK-30283` \
-      | grep -e "\[\s*ERROR\s*\]" || true)
+
+  logs="kubectl logs -n ${operator_pod_namespace} ${operator_pod_name}
```

Comment:
Maybe we can call it `cmd` which can be local.
Additionally it would be good some debug msg:
```
echo "Filter command: ${cmd}"
```

Reply:
added